tests/lib/rules/prefer-import-from-vue.js

```diff
@@ -22,7 +22,8 @@ tester.run('prefer-import-from-vue', rule, {
     `export * from 'vue'`,
     `import Foo from 'foo'`,
     `import { createApp } from 'vue'
-    export { createApp }`
+    export { createApp }`,
+    `import '@vue/runtime-dom'`
```

Comment:
To test the filename, define a test case as follows:

```suggestion
    {
      filename: 'test.d.ts',
      code: `import '@vue/runtime-dom'`
    }
```